packages/tests/components/SfTextarea/SfTextarea.PageObject.ts

```diff
@@ -0,0 +1,26 @@
+import { BasePage } from '../../utils/BasePage';
+
+export default class SfTextareaBaseObject extends BasePage {
+  isDisabled() {
+    this.container.should('be.disabled');
+    return this;
+  }
+
+  isNotDisabled() {
+    this.container.should('not.be.disabled');
+    return this;
+  }
+
+  hasReadonly() {
+    this.container.should('have.attr', 'readonly');
+  }
+
+  hasPlaceholder() {
+    this.container.should('have.attr', 'placeholder');
```

Comment:
You could also add assertion that check what is expected value of the placeholder.

Reply:
added